lib/node_modules/@stdlib/math/base/special/sech/include/stdlib/math/base/special/sech.h

```diff
@@ -0,0 +1,39 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef STDLIB_MATH_BASE_SPECIAL_SECH_H
+#define STDLIB_MATH_BASE_SPECIAL_SECH_H
+
+/*
+ * If C++, prevent name mangling so that the compiler emits a binary file having undecorated names, thus mirroring the behavior of a C compiler.
+ */
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    /**
+     * Computes the hyperbolic secant of a double-precision floating-point number.
+     */
+    double stdlib_base_sech(const double x);
```

Comment:
```suggestion
/**
* Computes the hyperbolic secant of a double-precision floating-point number.
*/
double stdlib_base_sech( const double x );
```

Adhere to project conventions.